client/src/app/site/pages/organization/pages/accounts/pages/account-detail/components/account-detail/account-detail.component.html

```diff
@@ -160,6 +160,10 @@ <h4>{{ 'Last email sent' | translate }}</h4>
         <mat-icon>security</mat-icon>
         <span>{{ 'Change password' | translate }}</span>
     </button>
+    <button mat-menu-item [routerLink]="['/', 'accounts', user?.id, 'meetings']">
+        <mat-icon>auto_awesome_mosaic</mat-icon>
+        <span>{{ 'Add to meetings' | translate }}</span>
+    </button>
```

Comment:
Shouldn't this also be in the container below since it is only executable with OML `can_manage_users`?